documentation/_puppetdb_nav.html

```diff
@@ -60,6 +60,7 @@ <h2 id="puppetdb">PuppetDB {{ puppetdb_version }}</h2>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/query.html">Query Structure</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/operators.html">Available Operators</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/paging.html">Query Paging</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/root.html">Root Endpoint (Experimental)</a></li>
```

Comment:
It actually is /pdb/query/v4 ... I'm pointing out 'Root Endpoint' here as this is just the name in documentation. I could have called it 'Index Endpoint' instead, and make the docs appear in index.html, but I just wasn't sure.